### PR TITLE
Add support for .NET Framework 4.6.1

### PR DESCRIPTION
**Pull request by FabienLavocat**
Add the support for .NET Framework 4.6.1 from GitHub Issue #4976

### CODE REVIEW

**Comment by barnson** [comment]
Thanks! For this and your wix3 PR, have you signed the [assignment agreement](http://wixtoolset.org/development/assignment-agreement/)?

**Comment by FabienLavocat** [comment]
No I haven't. The login link doesn't work. Do you have another link for me to register?

**Comment by rseanhall** [comment]
In wix4, new packages need to be documented at src/chm/documents/bundle/bundle_built_in_packagegroups.html.md.  Also, the BalExtension provides the packages that an MBA would use, so new packages need to also be created at src/ext/BalExtension/wixlib.

**Comment by FabienLavocat** [comment]
I've added the changes in the documentation and the BalExtension

**Comment by robmen** [comment]
@FabienLavocat The assignment page should be fixed now. Sorry for the delay.

**Comment by FabienLavocat** [comment]
Thanks! I've sent the agreement.

File: src/ext/BalExtension/wixlib/NetFx461AsPrereq.wxs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<!--
+  <copyright file="NetFx461AsPrereq.wxs" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'>
+    <?define NetFx461EulaLink = http://referencesource.microsoft.com/license.html ?>
```
Comment (robmen) [inline src/ext/BalExtension/wixlib/NetFx461AsPrereq.wxs:11]: This does not look like an official URL. Is there anything better?

**Comment by robmen** [comment]
Also, please add a history message (see history\readme.md) to get credit and have the fix show up in the WiX release feed.

**Comment by FabienLavocat** [comment]
Done! For Wix3 and Wix4

File: src/ext/BalExtension/wixlib/NetFx461AsPrereq.wxs (hunk 1 above)
Comment (FabienLavocat) [inline src/ext/BalExtension/wixlib/NetFx461AsPrereq.wxs:11]: Unfortunately, that's the only kind of license I've found for the .NET Fx

**Comment by alvarolorentedev** [comment]
Is there any news as if this will be merged into wix installer?, and if yes when?. I have a dependency on .net 4.6.1 and I would like to be able to use the net install version on the installer.

ps. ok i have been looking around and seems this is in v3 can anyone confirm? and also if there is going to be a weekly release any time soon?

**Comment by disklosr** [comment]
When is support for .Net 4.6.1 going to be released for Wix? I need it for one of my projects. Thanks.

**Comment by FabienLavocat** [comment]
@kanekotic @disklosr it has been merge already for Wix 3 but not Wix 4. I don't know when it will be in the official release.

**Comment by disklosr** [comment]
Tried with latest wix 3 release and it doesn't support .net 4.6.1.

**Pull request closed without merging**